docs/core/knowledge-bases.rst

```diff
@@ -37,7 +37,7 @@ custom action that inherits from ``ActionQueryKnowledgeBase``, a pre-written cus
 the logic to query a knowledge base for objects and their attributes.
 
 You can find a complete example in ``examples/knowledgebasebot``
-(`knowledge base bot <https://github.com/RasaHQ/rasa/blob/master/examples/knowledgebasebot/>`_), as well as instructions
+(`knowledge base bot <https://github.com/RasaHQ/rasa/tree/master/examples/knowledgebasebot/>`_), as well as instructions
```

Comment:
why is this converted to tree and the ones below to blob?

Reply:
I found that confusing too -- i believe it's because anything you find in the tree of searching for stuff (like the base project of the example) is a tree, while a file itself is a blob. Both work either way, just helps clean up the output a little with respect to redirects